dbt/adapters/athena/impl.py

```diff
@@ -240,6 +242,9 @@ def is_work_group_output_location_enforced(self) -> bool:
         conn = self.connections.get_thread_connection()
         creds = conn.credentials
 
+        if creds.work_group_enforced is not None:
+            return creds.work_group_enforced
```

Comment:
This return must be revisited,
> 
dbt/adapters/athena/impl.py:246: error: Returning Any from function declared to return "bool"  [no-any-return]